### DISCUSSION TITLE
uosc menu completely overwritten by "Stream quality"?

### DISCUSSION THREAD

**Discussion by xfzv** [christoph-heinrich/mpv-quality-menu#12] [Q&A] [answered]
When adding 

```
ctrl+f              script-binding quality_menu/video_formats_toggle #! Stream Quality > Video
alt+f               script-binding quality_menu/audio_formats_toggle #! Stream Quality > Audio
```

to my `input.conf`, my uosc menu is completely overwritten:

https://user-images.githubusercontent.com/78810647/193465882-e2caa436-1ba8-4055-a6df-d0d1ea75e70c.mp4

I guess I'm missing something, quoting the `README`: **Add** the video and audio menu to the uosc menu by appending...

Thanks in advance.

**Comment by christoph-heinrich** [comment, marked as answer]
That is because you don't have any other menu items specified in your input.conf.

uosc uses a default menu when nothing is specified, but by adding those two lines you specified something and so it doesn't use the default one anymore.

The one provided in [Examples](https://github.com/tomasklaen/uosc#examples) should be identical to the default menu. I recommend replacing the line `q           script-binding uosc/stream-quality     #! Stream quality` with the ones from my installation instructions.

**Reply by xfzv** [reply]
> but by adding those two lines you specified something and so it doesn't use the default one anymore.

That was it. Thank you very much!
